src/command_modules/azure-cli-botservice/HISTORY.rst

```diff
@@ -5,6 +5,10 @@ Release History
 
 0.1.7
 +++++
+* Add "SCM_DO_BUILD_DURING_DEPLOYMENT" to ARM template's Application Settings for v4 Web App Bots
```

Comment:
This should be put on top of `0.1.7`, as the `0.1.7` has been released

Reply:
Sorry about that! Fixed 😄